### PR TITLE
docs: formatting, GRUB duplicate entry removal

### PR DESCRIPTION
**Pull request by boredsquirrel**
this needs to stay manual for now, as it only works on UEFI properly

### CODE REVIEW

File: POSTINSTALL-README.md [hunk 1]
```diff
@@ -14,6 +15,15 @@ GRUB will prompt for a username and password. The default username is root.
 
 If you wish to password-protect booting existing entries, you can add the `grub_users root` entry in the specific configuration file located in the `/boot/loader/entries` directory.
 
+### Remove duplicate boot entries
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md:18]: This applies to all rpm-ostree systems, not just secureblue. This document is for additional _hardening_, not fixing upstream issues. 

So please remove this section. That said, you can leave in the other unrelated formatting improvements.

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

File: POSTINSTALL-README.md [hunk 2]
```diff
@@ -55,13 +65,14 @@ EOF
 
 The custom rule allows the group`diskadmin` to do the actions for unlocking and mounting these drives. Note the requirement on `active` and `local`, and the exactly specified actions.
 
-## Chromium Extension
+## Chromium
+### Extension
```
Comment (storopoli) [inline POSTINSTALL-README.md:69]: We should add some text between empty headers

File: POSTINSTALL-README.md [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 After rebasing to secureblue, the following steps are recommended.
 
-## Set a GRUB password
+## GRUB
+### Set a password
```
Comment (storopoli) [inline POSTINSTALL-README.md:6]: We should add some text between empty headers

**Comment by RoyalOughtness** [comment]
https://github.com/secureblue/secureblue/commit/848159d91c50ef78eaf6b3c5725df26b37b85f32

File: POSTINSTALL-README.md (hunk 3 above)
Comment (boredsquirrel) [reply POSTINSTALL-README.md:6]: I think that is a matter of taste? You could do that, but its not mandatory
